deploy/config/k8s/test-env-push-to-file.sh.yml

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+set -euo pipefail
+
+# Test cases can modify following variables from outside. Either by selecting
+# default valid input, by leaving it unmodified. Or setting a single key which
+# represent a key without a value. Or by setting it with invalid key or value.
+# Another option is to set it to $KEY_VALUE_NOT_EXIST which means to omit
+# related key-value from yaml file
+CONJUR_AUTHN_LOGIN=${CONJUR_AUTHN_LOGIN:-"host/conjur/authn-k8s/${AUTHENTICATOR_ID}/apps/${APP_NAMESPACE_NAME}/*/*"}
+#        conjur.org/authn-identity: 'host/conjur/authn-k8s/${AUTHENTICATOR_ID}/apps/${APP_NAMESPACE_NAME}'
```

Comment:
Eventually, we'll want to use the `conjur.org/authn-identity` annotation rather than the `CONJUR_AUTHN_LOGIN` environment variable, right? If we do use the annotation, it probably needs to have a value identical to the value used in the environment variable setting (i.e. should end with `/*/*`).

Reply:
yes, changed.